packages/visx-brush/src/BrushSelection.tsx

```diff
@@ -146,6 +148,15 @@ export default class BrushSelection extends React.Component<
               onClick={event => {
                 if (onClick) onClick(event);
               }}
+              onTouchStart={disableDraggingSelection ? undefined : dragStart}
```

Comment:
same comment on remove these additions and updating the above `mouse` handlers to the `pointer` equivalents 👍